sql/postgres/sqlspec_test.go

```diff
@@ -29,6 +29,12 @@ table "table" {
 	column "account_name" {
 		type = varchar(32)
 	}
+	column "varchar_length_is_not_required" {
+		type = varchar()
```

Comment:
```suggestion
		type = varchar
```

this will work but looks nicer as it will be written in the database